InternshipBE/Shared/Constants/ExportFileOffsets.cs

```diff
@@ -5,16 +5,27 @@ public static class ExportFileOffsets
         public const string INTERNSHIP_VALUE_OFFSET = "B1";
         public const string CANDIDATE_COUNT_VALUE_OFFSET = "B2";
         public const string CANDIDATE_COUNT_NAME_OFFSET = "A2";
-        public const string STYLE_RANGE_OFFSET = "A1:A2";
-        public const string STYLE_RANGE_ARRAY_NAME_OFFSET = "A4:C4";
```

Comment:
is it okey that you replace STYLE_RANGE_OFFSET = "A1:A2"; to STYLE_RANGE_OFFSET = "A1:A3";?